docs/configuration.md

```diff
@@ -8,7 +8,7 @@ The following configurations are available for users to customise:
 | aws.library.s3_bucket | S3 Bucket to upload/download AEM OpenCloud libraries to/from | Mandatory | |
 | aws.library.s3_path | S3 Path to the location of the AEM OpenCloud library artifacts | Optional | `library` |
 | aem_opencloud.version | AEM OpenCloud version value to be displayed on Jenkins folder with format `aem-opencloud-<version>` | Optional | The current AEM OpenCloud version |
-| aem_opencloud.config.artifact_url | URL to the location of the AEM OpenCloud configuration artifact | Mandatory | |
+| aem_opencloud.config.artifact_url | URL to the location of the AEM OpenCloud tar.gz archive configuration artifact | Mandatory | |
```

Comment:
Please also add similar note to about the file extension to other `*.artifact_url` config properties so they are consistent.